### ISSUE TITLE
Types are not generated

### ISSUE THREAD

**Issue by simplenotezy** [JasonShin/sqlx-ts#103]
I am having issues generating types from my project. I have successfully done it once, because I see a `my-file.queries.ts` file, however, I have tried deleting it again and rerunning: `npx sqlx-ts ./src --config .sqlxrc.json` - but nothing happens.

I have also tried running: `cargo run ./src/index.ts --config .sqlxrc.json` but that results in error: `error: could not find Cargo.toml in /Users/mf/Projects/project or any parent directory`

If I make a mistake the query, I will get an error, so it does indeed check the query, but it does not generate the types.

This is my config:

```json
{
  "generateTypes": {
    "enabled": true
  },
  "connections": {
    "default": {
      "DB_TYPE": "postgres",
      "DB_HOST": "localhost",
      "DB_PORT": 26257,
      "DB_USER": "root",
      "DB_PASS": "passwd",
      "DB_NAME": "defaultdb"
    }
  }
}
```

**Comment by simplenotezy** [comment]
Running:

```bash
yarn run sqlx-ts $(pwd) --generate-types --config .sqlxrc.json
```

Works - from here: https://github.com/JasonShin/sqlx-ts/issues/81

**Comment by simplenotezy** [comment]
Running: `npx sqlx-ts ./src --config .sqlxrc.json --generate-types` also works

**Comment by JasonShin** [comment]
Hello, we found a minor bug that `might` be related - the fix has been released in 0.12.1

Please try again and let me know

**Comment by simplenotezy** [comment]
Nice, thanks for the update @JasonShin
